rcaracaus/desarol
# 120: Blog Listing

Steps to test:

Go to https://desarol-pr-141.herokuapp.com/blog/ and you should see a listing of blogs
